airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java

```diff
@@ -987,7 +1001,7 @@ public UUID writeActorCatalogFetchEvent(final AirbyteCatalog catalog,
     final OffsetDateTime timestamp = OffsetDateTime.now();
     final UUID fetchEventID = UUID.randomUUID();
     return database.transaction(ctx -> {
-      final UUID catalogId = getOrInsertActorCatalog(catalog, ctx);
+      final UUID catalogId = getOrInsertActorCatalog(catalog, ctx, timestamp);
```

Comment:
this was a small change I made that isn't 100% related to the PR. When we create an actor_catalog_fetch_event, we sometimes create a new actor_catalog as well. This happens within a db transaction. I updated this so the actor_catalog_fetch_event & the actor_catalog that is created from that fetch event have the same created_at timestamp.

I previously though the timestamps would be needed for the logic to determine whether there is a new actor_catalog for a given source, but I think fetching the most recent one and checking whether it is the same is sufficient - but I left in this change anyway.